virttest/qemu_devices/qdevices.py

```diff
@@ -993,6 +994,109 @@ def verify_unplug(self, dev_type, monitor):
         return True
 
 
+@total_ordering
+class IOThread(QObject):
+    """iothread object.
+
+    attributes of iothread
+    ["id", "poll-max-ns", "poll-grow", "poll-shrink"]
+    """
+
+    def __init__(self, id, params=None):
+        """Constructor."""
+        if params is None:
+            params = dict()
+        params["id"] = id
```

Comment:
Better to use another name for the argument to distinguish with the built-in function `id`.